app/models/pull.rb

```diff
@@ -627,7 +629,7 @@ def commit_between
   end
 
   def base_branch_exists?
-    repository.branches.include? commit_base
+    repository.branches.include? commit_base if repository.branches
```

Comment:
The way it's written, `base_branch_exists?` could return `nil` witch doesn't make sense.

```suggestion
    repository.branches && repository.branches.include? commit_base
```

Reply:
Also, I think this fix should also be applied to `head_branch_exists?`